app/views/posts/index.html.erb

```diff
@@ -1,7 +1,37 @@
-<h1>All Posts for <%= @user.name %></h1>
+<div class="container">
+<div><div class="aside">
+    <div class="aside__left">
+    <div class="aside__left__responsive">
+    <img src= <%=  @user.photo %> alt= <%=  @user.id %> class="responsive aside__left__responsive__respo">
+    </div>
+    </div>
+    <div class="aside__right">
+    <%=  @user.name %>
+    <%=  @user.postsCounter %>
+    </div>
+    </div></div>
+</div>
 
-<ul>
-  <% @posts.each do |post| %>
-    <li><%= link_to post.title, user_post_path(@user, post) %></li>
+<ul class="useridposts">
+  <% @user.posts.each do |post| %>
+<div class="container">
+    <li class="postshow">
```

Comment:
- [ ] Let's use our `recent_five_comments` callback function to render the 5 most recent comments for a post. We do not need to hardcode the comments. You can use the rails console to test if this function is working. Create more than 5 comments on a post, and run the server to see if that post will only show 5 comments in the browser.